### ISSUE TITLE
Add AWS ElsaticCache for Redis as a vector store

### ISSUE THREAD

**Issue by simpleenigma** [theodo-group/LLPhant#26]
The Redis vector store uses features not available in AWS ElasticCache for Redis. Would it be possible to create a vector store for AWS ElasticCache for Redis?

**Comment by Spartee** [comment]
No, It's impossible to use Elasticache for this. You need Redis-Stack or Redis Cloud or Redis Enterprise.

**Comment by mortensi** [comment]
Redis 8 M01 is available for testing. From [Redis 8 on](https://redis.io/blog/redis-8-0-m01-released-one-redis-for-every-use-case/), the standard Redis Community Edition will support vector search.
